fms_core/resources/extraction.py

```diff
@@ -21,7 +21,7 @@
 
 
 class ExtractionResource(GenericResource):
-    biospecimen_type = Field(attribute='biospecimen_type', column_name='Extraction Type')
+    sample_kind = Field(attribute='sample_kind_name', column_name='Extraction Type')
```

Comment:
You are mapping to a non existing field here. You can remove the attribute if you are not using it to map to an existing field... You can test field.column_name instead below...